dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OkHttpUtils.java

```diff
@@ -35,53 +39,116 @@
 
 public class OkHttpUtils {
 
-    private static final OkHttpClient CLIENT = new OkHttpClient.Builder()
-            .connectTimeout(5, TimeUnit.MINUTES) // connect timeout
-            .writeTimeout(5, TimeUnit.MINUTES) // write timeout
-            .readTimeout(5, TimeUnit.MINUTES)
-            .build();
-
-    public static @NonNull String get(@NonNull String url,
-                                      @Nullable Map<String, String> httpHeaders,
-                                      @Nullable Map<String, Object> requestParams) throws IOException {
+    private static OkHttpClient CLIENT = new OkHttpClient();
+
+    public static @NonNull OkHttpResponse get(@NonNull String url,
```

Comment:
Please add some method description, especially some units of time. 

Same for other methods.

Reply:
Fixed.